### ISSUE TITLE
Update MS Teams webhook URLs

### ISSUE THREAD

**Issue by jemnery** [ministryofjustice/find-moj-data#1000]
There is currently a deprecation warning in our MS Teams deployment [notification](https://teams.microsoft.com/l/message/19:2053a61bc90c4012aef09f5afe998ac8@thread.tacv2/1730118091593?tenantId=c6874728-71e6-41fe-a9e1-2e8c36776ad8&groupId=f6c3cb3b-591c-4e47-9997-25b6dc9bf5b6&parentMessageId=1730118091593&teamName=Data%20Catalogue&channelName=Prod%20Deployment%20Notifications&createdTime=1730118091593).

[This article](https://learn.microsoft.com/en-gb/microsoftteams/m365-custom-connectors#update-connectors-url) describes the required steps, but it's essentially:

- [x] Generate new webhook URLs in MS Teams ([preprod](https://teams.microsoft.com/l/channel/19%3Ad97678bddb074a46b7ce3eae3f825cd6%40thread.tacv2/Preprod%20Deployment%20Notifications?groupId=f6c3cb3b-591c-4e47-9997-25b6dc9bf5b6&tenantId=c6874728-71e6-41fe-a9e1-2e8c36776ad8) and [prod](https://teams.microsoft.com/l/channel/19%3A2053a61bc90c4012aef09f5afe998ac8%40thread.tacv2/Prod%20Deployment%20Notifications?groupId=f6c3cb3b-591c-4e47-9997-25b6dc9bf5b6&tenantId=c6874728-71e6-41fe-a9e1-2e8c36776ad8) channels)
- [x] Update relevant app secrets
- [x] Re-deploy and check the message no longer has the deprecation warning

----

![Image](https://github.com/user-attachments/assets/45e75664-76d8-4872-a0d7-56d4b0823b62)

**Comment by mitchdawson1982** [comment]
In the course of this ticket I have found a couple of issues with the pre prod notification workflow. 

- The casing of the status variable was incorrectly cased hence the notification was not triggered on success, Addressed here - https://github.com/ministryofjustice/find-moj-data/pull/1083
- There is no logic to define a successful run of the pre prod workflow, the success variable is statically configured rather than checking the status of the workflow. I will create a ticket to address this.
- Since updating the MS Teams webhook url, the channel no longer receives messages. I have reviewed [posts](https://techcommunity.microsoft.com/discussions/microsoftteams/webhook-issue-with-ms-teams/3971915) with the same symptoms and have removed and recreated the webhook however the issue remains.
